lib/cog_api/client.ex

```diff
@@ -46,6 +46,8 @@ defmodule CogApi.Client do
   @callback relay_group_delete(String.t, %Endpoint{}) :: {atom, [String.t]}
   @callback relay_group_add_relay(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
   @callback relay_group_remove_relay(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
+  @callback relay_group_add_bundle(String.t, String.t, %Endpoint{}) :: {atom, %RelayGroup{}}
```

Comment:
There is also an `add_bundle` that accepts a `%{name: bundle_name}` on the `HTTP` side.